modules/packages/MPI.chpl

```diff
@@ -29,9 +37,10 @@ routines ::
       MPI_Keyval_create
       MPI_Keyval_free
 
-since all of these are built around user-defined handlers, that we do not support.
+since all of these are built around user-defined handlers, that is not supported.
```

Comment:
perhaps:

"since all of these are built around user-defined handlers that we do not support."